packages/messaging/src/Message.spec.ts

```diff
@@ -1308,14 +1334,12 @@ describe('Error checking / Verification', () => {
     expect(() => Message.verifyMessageBody(rejectTermsBody)).toThrowError(
       SDKErrors.CTypeHashMissingError
     )
-    requestAttestationBody.content.credential.claimerSignature = {
+    submitCredentialBody.content[0].claimerSignature = {
```

Comment:
This is because `requestAttestationBody` wouldn't need a signature now, correct?

Reply:
exactly, this property is now unknown